app/models/concerns/search.rb

```diff
@@ -40,6 +26,17 @@ def status(param)
       param == 'active' ? where(active: true) : where(active: false)
     end
 
+    def keyword(k)
+      tsquery = <<-SQL
+        (to_tsquery('english', ''' ' || '#{k}' || ' ''' || ':*'))
```

Comment:
Ha! I was just gonna say the same thing. I wonder why. From what I can tell, all it does is place quotes around the query, which is the same as what I have now: `'#{k}'`.